src/Cookie/FileCookieJar.php

```diff
@@ -79,7 +79,8 @@ public function load(string $filename): void
         $json = \file_get_contents($filename);
         if (false === $json) {
             throw new \RuntimeException("Unable to load file {$filename}");
-        } elseif ($json === '') {
+        }
+        if ($json === '') {
```

Comment:
Let's avoid these kind of changes.. as 6.5 is still active and it will make merging back from 6.5 difficult

Reply:
6.5 branch is barely active now. I'd be happy to handling merging 6.5 -> 7.x going forward, in any case.

Reply:
Fair enough.. and doing a quick pass this might be the only place... we need to get back to the php-cs rules thing.. #2498